### PR TITLE
Add additional contributors

### LINKED ISSUE #58: attribute more contributors or editors
I want more people credited on this spec than who the spec currently mentions. This is A) 100% true; and B) politically desirable. Would people mind if I submitted a PR putting other names in -- and if not, who wants to be an editor vs. a contributor?

Tagging a few people who might need to be named. @2byrds @m00sey @peacekeeper @talltree @nkongsuwan @SmithSamuelM

### PR DESCRIPTION
**Pull request by dhh1128**
Fixes #58

### CODE REVIEW

**Review by 2byrds — APPROVED** [review]
LGTM

**Review by swcurran — APPROVED** [review]
Thanks Daniel.

**Comment by dhh1128** [comment]
@nkongsuwan , do you want to be credited?

**Comment by nkongsuwan** [comment]
@dhh1128 Yes, I would love to be credited. However, I am not quite sure if I have made sufficient contributions to be credited as I have so far taken more of an observer role.

File: spec/header.md [hunk 1]
```diff
@@ -10,8 +10,16 @@ DID Method `did:webs` Specification
...
 - [Phil Feairheller](https://github.com/pfeairheller), [GLEIF](https://gleif.org)
-- [Daniel Hardman](https://www.linkedin.com/in/danielhardman/), [Provenant, Inc](https://provenant.net), W3CID: 108790
-- [Stephen Curran](https://github.com/swcurran)
+- [Daniel Hardman](https://www.linkedin.com/in/danielhardman/), [Provenant, Inc](https://provenant.net)
+- [Stephen Curran](https://github.com/swcurran), [Cloud Compass](https://cloudcompass.ca/)
+- [Sam Smith](https://github.com/SmithSamuelM), [Prosapien](https://prosapien.com/)
+
+**Contributors:**
+
+- [Markus Sabadello](https://www.linkedin.com/in/johnjordandigitaltrust/), [Danube Tech](https://danubetech.com/) and [DIF](https://identity.foundation)
```
Comment (peacekeeper) [inline spec/header.md:19]:
```suggestion
- [Markus Sabadello](https://www.linkedin.com/in/markus-sabadello-353a0821/), [Danube Tech](https://danubetech.com/) and [DIF](https://identity.foundation)
```

**Review by peacekeeper — APPROVED** [review]
Fix my LinkedIn, otherwise looks good!

**Comment by dhh1128** [comment]
@nkongsuwan : you read the spec carefully enough to catch a subtle error in the technicalities, and you fixed it. Thank you. I think that counts as contribution.

**Pull request merged**